types/styled-system__should-forward-prop/index.d.ts

```diff
@@ -0,0 +1,10 @@
+// Type definitions for @styled-system/should-forward-prop 5.1
+// Project: https://github.com/styled-system/styled-system/tree/master/packages/should-forward-prop
+// Definitions by: Tom Picton <https://github.com/tpict>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+type genericShouldForwardProp = (prop: string) => boolean;
```

Comment:
Conventionally, types are PascalCased, but since this isn’t exported it’s not a big deal.